src/twisted/internet/_sslverify.py

```diff
@@ -1490,6 +1598,11 @@ def __init__(self,
             earlier in the list are preferred over those later in the list.
         @type acceptableProtocols: L{list} of L{bytes}
 
+        @param tlsProtocols: The TLS protocols this peer wishes to speak.
+            Either a single protocol from L{TLSVersion} or a L{TLSVersionRange}
+            with a lowest and highest L{TLSVersion}.
+        @type tlsProtocols: L{TLSVersion} constant or L{TLSVersionRange}
```

Comment:
Eww?  Don't do this, please.  If people want to be able to specify a single version, replace `TLSVersionRange` with something that doesn't make doing that so complicated that you're tempted to have a parameter that is either a scalar or a vector.